src/main/java/org/jabref/gui/search/GlobalSearchBar.java

```diff
@@ -284,11 +285,14 @@ public void updateResults(int matched, TextFlow description, boolean grammarBase
             // TODO: switch Icon color
             //searchIcon.setIcon(IconTheme.JabRefIcon.SEARCH.getIcon());
         }
-        Tooltip tooltip = new Tooltip();
-        tooltip.setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
-        tooltip.setGraphic(description);
-        tooltip.setMaxHeight(10);
-        searchField.setTooltip(tooltip);
+
+        if (Globals.prefs.getBoolean(JabRefPreferences.SHOW_ADVANCED_HINTS)) {
+            Tooltip tooltip = new Tooltip();
+            tooltip.setContentDisplay(ContentDisplay.GRAPHIC_ONLY);
+            tooltip.setGraphic(description);
+            tooltip.setMaxHeight(10);
+            searchField.setTooltip(tooltip);
+        }
```

Comment:
Please add these lines of code to prevent the last tooltip from displaying again after the user unchecks this preference. Thanks!

```
        else {
            searchField.setTooltip(null);
        }
```

Reply:
![tooltip](https://user-images.githubusercontent.com/12954316/57969018-16f68c80-7972-11e9-9351-70c95e158235.gif)

Reply:
Thanks for your note.
I was just refactoring this method a little bit, creating a seperate method (updateTooltip), since there is no tooltip (and no hint) at all, when jabref just started. I will pick it up.